sec_certs_page/templates/fips/data.html.jinja2

```diff
@@ -0,0 +1,68 @@
+{% from "common/library-info.html.jinja2" import render_library_info %}
+{% set crumbs = False %}
+{% set search = False %}
+{% set canonical = True %}
+{% block title %}
+    <title>FIPS 140 | sec-certs.org</title>
+{% endblock %}
+
+{% block content %}
+    <main>
+       <div class="container pt-4">
+            {{ render_library_info() }}
+            <h3 class="anchor" id="data">Data</h3>
+            <p>
+                The full dataset used on this site can be downloaded here. It is updated weekly
+                and all the pages and data on this site are extracted from it. It is a JSON file as produced
+                by the <code>sec-certs fips</code> command from the tool. Documentation for the tool,
+                including a quick-start guide, can be found
+                <a href="{{ url_for("docs.serve_docs", path="index.html") }}">here</a>.
+            </p>
+            {% if last_ok_run %}
+                <p>
+                    The dataset was last successfully updated
+                    at {{ last_ok_run["start_time"] | strftime("%H:%M on %d.%m.%Y") }}
+                    using version <span class="font-monospace">{{ last_ok_run["tool_version"] }}</span> of the
+                    sec-certs tool
+                    and it contains {{ last_ok_run["length"] }} certificates.
+                </p>
+            {% endif %}
+            <table class="spaced-table align-middle table table-borderless">
+                <tr>
+                    <td>
+                        <a href="{{ url_for(".dataset") }}" role="button" class="btn btn-primary w-100"><i
+                                class="fas fa-file-alt"></i>
+                            Main dataset</a>
+                    </td>
+                    <td>
+                        The main dataset is a JSON file with data on the certificates extracted from the
+                        certification artifacts (i.e. PDFs). It does not contain the files themselves.
+                    </td>
+                </tr>
+                <tr>
+                    <td>
+                        <a href="{{ url_for(".dataset_archive") }}" role="button" class="btn btn-primary w-100">
+                            <i class="fas fa-file-archive"></i> Full datasset
+                        </a>
+                    </td>
+                    <td>
+                        The full dataset contains the dataset mentioned above as well as the certification
+                        artifacts themselves (i.e. PDFs). It is a gzip-compressed tar archive.
+                        Download this if you want to do your own processing on top of the data provided by our
+                        tooling.
+                    </td>
+                </tr>
+            </table>
+            <div class="row p-3 py-md-5">
+                <h3>Data sources</h3>
+                <p>
+                    We are getting information from the following data sources:
+                </p>
```

Comment:
This is empty, whaat is the plan here?

Reply:
It is up for a discussion. We have got this list of data sources: https://docs.google.com/document/d/1MJ2aSAnMlWZLNpui2XzCV8BwO02Xp8imvoI6PZI-Pno/edit?tab=t.0

Do you think that we should put it on a website? If so, CC and PP data sources will be identical and Others will be in Vulnerabilities section?

Reply:
We could include those. Do you want to put them in there on the web or should I have a go?

Reply:
I don't have too much time before final exams, so I would like to finish this MR as soon as possible. So we can either exclude this from MR or if you would like to, you can add it.